updateJobs-fromTemplateA2TemplateB.groovy

```diff
@@ -0,0 +1,33 @@
+/*
+Author: Carlos Rodriguez Lopez
+Since: July 2017
+Description: It finds all the jobs in CJE instance with  "Template_A"  and then update them to "Template_B"
+Scope: Cloudbees Jenkins Enterprise
+*/
+
+import com.cloudbees.hudson.plugins.modeling.ModelList
+import com.cloudbees.hudson.plugins.modeling.impl.jobTemplate.InstanceFromJobTemplate
+import com.cloudbees.hudson.plugins.modeling.impl.jobTemplate.JobPropertyImpl
+
+def newJobTemplateName = 'Template_B'
+def oldJobTemplateName = 'Template_A'
+def numberOfJobUpdated = 0
+
+Jenkins.instance.items.findAll { job ->
+    def action = job.getAction(com.cloudbees.hudson.plugins.modeling.impl.entity.LinkToTemplateAction.class)
+    if (action != null) {
+        if (action.instance.model.name == oldJobTemplateName) {
+            //Create a a Instance from the Job Template "newJobTemplateName"
+            def inst = new InstanceFromJobTemplate(ModelList.get().getItem(newJobTemplateName))
+            println "Job `${job.name}` currently using the `${action.instance.model.name}` template"
+            //Assigning new template and saving it
+            job.addProperty(new JobPropertyImpl(inst))
```

Comment:
This will add a duplicate property I think.

It is safer and higher-level to use `EntityModel.instantiate`, which is what the REST and CLI commands use.